### DISCUSSION TITLE
Stuck in "Revisiting"

### DISCUSSION THREAD

**Discussion by alandaitch** [omkarcloud/google-maps-scraper#62] [General]
how do I solve this?

Creating Driver with window_size=1920,1080 and user_agent=Mozilla/5.0 (Macintosh; Intel Mac OS X) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36
Launched Browser
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting
Revisiting

**Comment by Chetan11-dev** [comment]
Please wait till 18th November, will be releasing a fix for it.

**Comment by Chetan11-dev** [comment]
The issue is fixed. Please re-clone the repository and run it. Remember to execute 
```
python -m pip install -r requirements.txt
```
